scripts/mirror_releases_zot.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
add some doc or a printf at the end so the user knows what to do with the stdout?